documentation/content/Supported Plugins/multi-part-plugin.md

```diff
@@ -1,69 +1,70 @@
-Title: How to use Multi Part plugin
+Title: Add a Series Indicator to Your Multipart Articles
```

Comment:
Instead of "Add a series indicator" we should make it clear it's about series plugin.

Reply:
changed it to "Displaying a Series Information for Your Multipart Articles"  I would prefer to focus on the action they want to take instead of the plugin.  I.e. I looked on the main Elegant site, and notice they have this section that shows up for multipart articles.  How do I do that?

Thoughts?